### ISSUE TITLE
createdAt as an argument to bs_post?

### ISSUE THREAD

**Issue by grasshoppermouse** [christopherkenny/bskyr#21]
I want to use this package to transfer my tweets to Bluesky using the original tweet date and time. Any interest in making `createdAt` an argument to `bs_post`, with a default?:

`bs_post(..., createdAt = bs_created_at())`

**Comment by christopherkenny** [comment]
This would be very easy to add. My reason for not doing so is that posts are intended to be timed in a "true" manner. Posts with a `createdAt` date very different from their `indexedAt` date (the time that the server first sees the post) are flagged with a warning.

That said, I'm happy to add the argument if it's helpful with that warning in mind.

**Comment by grasshoppermouse** [comment]
I agree with the archived flag that Bluesky adds on backdated posts. In fact, I think that flag makes it safe to add this argument, i.e., it's a supported feature of Bluesky.

The ability to import tweets into Bluesky with their original post date would be incredibly helpful.

**Comment by christopherkenny** [comment]
This should work now. Beware that it gets upset if it isn't formatted as "%Y-%m-%dT%H:%M:%OS6Z". To make it happy, follow the format currently used in `bs_created_at()`:

```
bs_created_at <- function() {
  format(lubridate::now('UTC'), format = '%Y-%m-%dT%H:%M:%OS6Z')
}
```
